webapp/channels/src/packages/mattermost-redux/src/actions/posts.ts

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+/* eslint-disable max-lines */
```

Comment:
I forget if you mentioned yesterday that you wanted to remove this or not before merging the PR

Reply:
removed